qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -356,10 +362,11 @@ def verify_removed(snaps_removed):
         self.mount_a.run_shell(['rmdir', TestSnapSchedules.TEST_DIRECTORY])
 
     def get_snap_stats(self, dir_path):
-        snap_path = f"{dir_path}/.snap"[1:]
+        sdn = self.fs.get_config('client_snapdir')
+        snap_path = f"{dir_path}/{sdn}"[1:]
```

Comment:
Suggestion: All strings are enclosed in single quotes, would be better to maintain this uniformity.

Reply:
Allowed to use any quote for f-string literals but we can have for this file,atleast to maintain uniformity.

Reply:
We can, It's just that I was looking at other changes that you made in this file and as you have used single quotes everywhere, I thought it could be better to have single quotes here too, but yeah it's not violating anything so it's fine.